marimo/_cli/development/commands.py

```diff
@@ -14,21 +17,82 @@
 import marimo._server.models.models as models
 import marimo._snippets.snippets as snippets
 from marimo import __version__
-from marimo._ast.cell import CellConfig
+from marimo._ast.cell import CellConfig, CellStatusType
 from marimo._config.config import MarimoConfig
+from marimo._messaging.cell_output import CellChannel, CellOutput
+from marimo._messaging.mimetypes import KnownMimeType
+from marimo._output.mime import MIME
+from marimo._plugins.core.web_component import JSONType
 from marimo._server.api.router import build_routes
-from marimo._utils.dataclass_to_openapi import dataclass_to_openapi_spec
+from marimo._utils.dataclass_to_openapi import (
+    python_type_name,
+    python_type_to_openapi_type,
+)
 
 
 def _generate_schema() -> dict[str, Any]:
-    # dataclass components that we can generate schemas for
-    COMPONENTS = [
+    # dataclass components used in websocket messages
+    # these are always snake_case
```

Comment:
Aren't these CamelCase?

Reply:
they are not, they are snake case. and not sure we can migrate to cameCase since the `data` attribute on CellOutput would be converted too. 

i think we could if we make `deep_camel_case` only operate on dataclass fields

Reply:
Oh, I was just looking at the class names, but the field names are snake case, which is what you must be referring too. Got it, makes sense